src-plugins/vtkDataMesh/writers/vtkDataMeshWriter.cpp

```diff
@@ -54,6 +54,9 @@ bool vtkDataMeshWriter::write(const QString& path)
 
   try
   {
+      setlocale(LC_NUMERIC, "C");
+      QLocale::setDefault(QLocale("C"));
```

Comment:
Nice. I tried to do this but in 'higher' in the application code and it did not work :(

Reply:
Good thing if this fixes the bug, but we should remember to remove it later (when the bug is gone) because this sort of code should only be in the main files.

Reply:
I remember something about plugins not keeping the locale set in the main app. I had the problem with medPipeline. Is it possible to test by putting this in the initialization of this plugin ?

Reply:
(actually I remember now: it was to do with text codecs but it's a similar issue)

Reply:
Not keeping text codecs is what was causing the problem with loading file with accent in their path

Reply:
(it does not work in the plugin. @fcollot validates, i merge)